src/context/src/lib.rs

```diff
@@ -30,10 +30,12 @@ use std::path::Path;
 
 pub mod video_context;
 pub mod broadcaster;
+pub mod timing;
 pub mod event;
 use video_context::{VideoContext, DisplayConfig};
 use broadcaster::Broadcaster;
 use event::EngineEvent;
+use timing::{Duration, SteadyTime};
```

Comment:
You are right. It was a bit hidden. I am going to test out the rest tonight and this weekend. Good work.